packages/config/__snapshots__/index.spec.ts.js

```diff
@@ -218,6 +220,7 @@ exports['config/src/index .getPublicConfigKeys returns list of public config key
   'experimentalRunAllSpecs',
   'experimentalMemoryManagement',
   'experimentalModifyObstructiveThirdPartyCode',
+  'injectDocumentDomain',
   'experimentalSkipDomainInjection',
```

Comment:
this should be removed from the snapshots
```suggestion
  '',
```